scripts/gn/download_rinex_deps.py

```diff
@@ -36,72 +43,42 @@ def _download_static_dependencies(start_date: _date, end_date: _date, target_dir
     :param end_date: Last date to download for
     :param target_dir: Target directory to download files to
     """
-    for date in daterange(start_date, end_date):
-        files = _get_static_long_filenames(date)
-
-        gps_week = dt2gpswk(date)
-        ftp_folder = f"gnss/products/{gps_week}"
-
-        download_multiple_files_from_cddis(files, ftp_folder, target_dir)
-
-
-def _get_static_long_filenames(date: _date) -> [str]:
-    """Deterimines the long filenames for sp3, erp and clk files for a given date,
-    which will allow ginan to run ppp static.
-
-    :param date: Date to determine filenames for
-    :returns [str] of filenames
-    """
-    # TODO: Support more than just IGS FIN products
-
-    # Weekly files
-    week = timedelta(7)
-
-    def _first_day_of_week(date: _date):
-        return date + timedelta(days=-(date.weekday() + 1))
-
-    first_day_of_week = _first_day_of_week(date)
-    erp_filename = generate_IGS_long_filename(
+    auto_download(
+        target_dir,
+        preset=None,
+        station_list=None,
+        start_datetime=start_date.strftime("%Y-%m-%d"),
+        end_datetime=end_date.strftime("%Y-%m-%d"),
+        replace=True,
+        dont_replace=False,
+        most_recent=False,
         analysis_center="IGS",
-        content_type="ERP",
-        format_type="ERP",
-        start_epoch=first_day_of_week,
-        timespan=week,
+        atx=False,
+        aload=False,
+        igrf=False,
+        egm=False,
+        oload=False,
+        opole=False,
+        fes=False,
+        planet=False,
+        sat_meta=False,
+        yaw=False,
+        snx=False,
+        nav=False,
+        sp3=True,
+        erp=True,
+        clk=True,
+        bia=False,
+        gpt2=False,
```

Comment:
You'll probably want to have this one on? Unless you grab this file from elsewhere?